hw_02/Task1/Program.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //явное преобразование типов
```

Comment:
I would suggest do not use Russian comments. try to use English in a code every time